docs/development-process/standard-release-process.md

```diff
@@ -0,0 +1,53 @@
+# Standard Release Process
+
+The following document is a description of the standard release process of
+Zarhus-related images. Precise steps and differences from the standard
+process will be described in layer-specific documentation.
+
+## Process steps
+
+1. Make sure that everything that should go into the given release is merged to
+   into main.
```

Comment:
@PLangowski we should emphasize variables, branch names etc.
```suggestion
   into `main`.
```

Reply:
https://github.com/zarhus/docs/compare/20e8d1eb800f1e034ce33d1d4bbe2617af90d8a1..9a81040be435abcac59cb8fd58b28a2cc01adf3d#diff-601babdf30d07c9e8dc632e099bf77c53afe5e0ec2dc70a2d37ea0cbfdbe3c5cR10